### PR TITLE
Minor tweaks

### PR DESCRIPTION
**Pull request by supercrafter100**
- Rewrite system prompt to make it include the classes it writes (overkill for what we're doing but nice to have)
- Make the writer pause slightly longer when its writing punctuation characters (as this is usually the case for humans as well)

### CODE REVIEW

File: src/util/realisticTyping.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const delays = {
   deleteRange: 200,
   moveCursor: 100,
   applyDiff: 100,
+  punctuationDelay: 150,
```
Comment (dipamsen) [inline src/util/realisticTyping.js:9]:
```suggestion
  typePunctuation: 150,
```
perhaps for consistent naming (to distinguish actions `type`, `delete`, `move`)

File: src/prompt.txt [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+- You will write the code in a code block and narrate your process explaining the parts. This means you are going to iterate multiple times, improving your code every step. You will include these iterations in your initial response however. 
+- You will say what you are going to write before writing it.
+- You will note the code you are writing with ```.
+- You will write ALL the code in the SAME file. In this case the sketch.js. This means ALL classes and functions will be in the SAME code you return for each step. You will NOT split them up into multiple files or code blocks.
+- You will ALWAYS provide the FULL SKETCH for each step. This means you will include the code you've written in the previous steps in the current step you're explaining. Including the classes of the file. You don't have to include code for next steps before you did those steps.
+- You will not write the language you are using after the ```.
+- If you are told to continue, add or modify something, you will ALWAYS include the current the code you've written so far in your response when you are adding onto it. As well as including any classes you've written.
```
Comment (shiffman) [inline src/prompt.txt:12]: We can probably remove the classes part for now but this is a good reference point for later!

File: src/prompt.txt [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
+- You will NOT write ANY comments in your code.
+- You will make errors and mistakes in your code on purpose every now and then after which you can correct them.
```
Comment (shiffman) [inline src/prompt.txt:14]: I find that LLMs don't work super well with "every now and then" kinds of prompts, they don't really retain a sense of how long they've been operating and the idea of doing a task with some about of frequency.

**Review by shiffman — COMMENTED** [review]
This is great! I'll test tomorrow when I get to practicing!

File: src/commands/listenToHuman.js [hunk 4]
```diff
@@ -60,12 +69,14 @@ class ListenToHumanCommand extends Command {
       recordProgram: 'sox',
     });
     this.recording.stream().pipe(file);
+    this.statusBarItem.text = '$(unmute) Currently listening';
     vscode.window.showInformationMessage('Listening...');
```
Comment (dipamsen) [inline src/commands/listenToHuman.js:73]: can be removed now?

File: src/commands/listenToHuman.js [hunk 5]
```diff
@@ -60,12 +69,14 @@ class ListenToHumanCommand extends Command {
...
     this.listening = true;
   }
 
   async stopListening() {
     this.recording.stop();
+    this.statusBarItem.text = '$(mute) Not listening';
     vscode.window.showInformationMessage('Stopped listening...');
```
Comment (dipamsen) [inline src/commands/listenToHuman.js:80]: same

**Pull request merged**